extensions/mailer/deployment/src/main/java/io/quarkus/mailer/deployment/MailerProcessor.java

```diff
@@ -11,7 +11,7 @@
 import io.quarkus.deployment.builditem.LaunchModeBuildItem;
 import io.quarkus.deployment.builditem.ShutdownContextBuildItem;
 import io.quarkus.deployment.builditem.substrate.ReflectiveClassBuildItem;
-import io.quarkus.mailer.impl.*;
+import io.quarkus.mailer.runtime.*;
```

Comment:
Does not look related but I think  we can avoid using start imports here :-)

Reply:
Yeah, it was preexisting but I pushed an update to fix it.